std_msgs/Header header

int32 id
geometry_msgs/Pose floor_center
int32[] keyframe_ids